Sources/StreamChatSwiftUI/CommonViews/SearchBar.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import SwiftUI
+
+struct SearchBar: View {
+    
+    @Injected(\.colors) private var colors
+    @Injected(\.fonts) private var fonts
+    @Injected(\.images) private var images
+    
+    @Binding var text: String
+    @State private var isEditing = false
+        
+    var body: some View {
+        HStack {
+            TextField(L10n.Message.Search.title, text: $text)
+                .padding(8)
+                .padding(.leading, 8)
+                .padding(.horizontal, 24)
+                .background(Color(colors.background1))
+                .cornerRadius(18)
```

Comment:
I think the corners look a bit off. I think with that it looks better, but that might also be personal preference:

```suggestion
                .clipShape(RoundedRectangle(cornerRadius: 18, style: .continuous))
```